spec/controllers/idv/verify_info_controller_spec.rb

```diff
@@ -491,4 +491,36 @@
       end
     end
   end
+
+  describe '#best_effort_phone' do
```

Comment:
@matthinz I did end up adding this in the controller spec based on your feedback. I set out to add a test for VerifyInfoConcern but it was turtles all the way down.